lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -212,7 +212,7 @@ static llvm::Expected<unsigned_t> getChannel(const ArgumentDictionaryTy &dict) {
   if (orderIt != dict.end()) {
     ASSIGN_VALUE_OR_RETURN_ERR(order, loadStr(orderIt->second));
   }
-  if (order == "NHWC") {
+  if (order == GLOW_DEFAULT_4D_LAYOUT) {
```

Comment:
Please use a concrete string here, because any change to the definition of GLOW_DEFAULT_4D_LAYOUT would break this code.